docs/react-testing-library/api.mdx

```diff
@@ -370,6 +370,30 @@ test('returns logged in user', () => {
 })
 ```
 
+If you have multiple props, try the following recipie
+```jsx
+function useMyGreetingHook(userA, userB) {
+  const greeting = React.useMemo(() => {
+    return 'Hello ' + userA + ' and ' + userB
+  }, [userA, userB])
+  return greeting
+}
+
+describe('useMyGreetingHook', () => {
```

Comment:
Instead of creating a new example, wouldn't it be simpler to add the example to the existing snippet?
E.g. with the following:

```
rerender({name: 'Bob'})
expect(result.current).toEqual({name: 'Bob'})
```

Reply:
cleaning up original example is fine too

ps. Examples in hooks section are a bit difficult to discern :(  
https://github.com/testing-library/react-hooks-testing-library/issues/743